### DISCUSSION TITLE
Support for old search operators, missing repos?

### DISCUSSION THREAD

**Discussion by jr-bf** [community/community#9952] [Other Feature Feedback, Questions, & Ideas] [answered]
In my day-to-day work doing security testing, I do a lot of searching for public repos outside my org that have 0 stars, 0 forks, 0 watches. I'm seeing a lot of these that are included in normal search, but not indexed by Code Search.
I understand these may simply be missing due to this being a feature preview, but was wondering if there are any plans to support search across _all_ repos or using old search operators like `stars:>1000`?

**Comment by look** [comment, marked as answer]
Thanks for the suggestion. The old code search doesn't support a `stars:` qualifier ([documentation](https://docs.github.com/en/search-github/searching-on-github/searching-code)), but it could be useful.

We are adding more repositories every day, so the gap between what's indexed in the old code search and the technical preview is narrowing.
